components/frontdoor/main.tf

```diff
@@ -24,33 +24,7 @@ locals {
   vault_name              = "acme${replace(local.subscription_short_name, "-", "")}"
 }
 
-module "landing_zone" {
-  count  = var.upgrade_frontdoor ? 0 : 1
-  source = "git::https://github.com/hmcts/terraform-module-frontdoor.git?ref=master"
-
-  common_tags                = module.ctags.common_tags
-  env                        = var.env
-  project                    = var.project
-  location                   = var.location
-  frontends                  = var.frontends
-  ssl_mode                   = "FrontDoor"
-  resource_group             = azurerm_resource_group.fd_rg.name
-  subscription_id            = data.azurerm_subscription.current.subscription_id
-  certificate_key_vault_name = local.vault_name
-  oms_env                    = var.oms_env
-  certificate_name_check     = true
-  key_vault_resource_group   = "sds-platform-${var.environment}-rg"
-  log_analytics_workspace_id = module.logworkspace.workspace_id
-
-  diagnostics_storage_account_id    = azurerm_storage_account.diagnostics.id
-  send_access_logs_to_log_analytics = false
-}
-moved {
-  from = module.landing_zone
-  to   = module.landing_zone[0]
-}
```

Comment:
this needs restoring in the inverse direction

Reply:
yupp